components/Contacts.vue

```diff
@@ -3,8 +3,8 @@
     <h3 class="card__title">{{ $t('Contacts', locale) }}</h3>
     <address :key="contact.sys.id" v-for="contact in content" class="card__contact contact">
       <h4 class="name">{{ contact.fields.name }}</h4>
-      <span class="job-title">{{ contact.fields.jobTitle }}</span><br>
-      <a class="email" :href="'mailto:' + hideEmail ? 'no-bots@example.com' : contact.fields.email">{{ hideEmail ? 'no-bots@example.com' : contact.fields.email }}</a>
```

Comment:
`'mailto:' + hideEmail` will always be true since it's string concatenation. Needed some parens.